torch/distributed/rpc/api.py

```diff
@@ -612,7 +612,7 @@ def remote(to, func, args=None, kwargs=None, timeout=UNSET_RPC_TIMEOUT):
 
         >>> # On both workers:
         >>> @torch.jit.script
-        >>> def my_script_add(t1, t2):
+        >>> def my_script_add(t1: torch.Tensor, t2: int):
```

Comment:
changing `t2`->`num` as @awgu suggested sounds good.

Reply:
Thanks for the suggestion, @H-Huang. I was also considering renaming the variables to tensor and scalar for even clearer context. Do you think that might be a more descriptive approach? Would love to hear your thoughts.

Reply:
Naming them tensor and scalar sounds good to me.